### PR TITLE
WIP: New eval quickstart

### PR DESCRIPTION
**Pull request by davidx33**
new Quickstart for evaluations

### CODE REVIEW

File: docs/evaluation/index.mdx [hunk 1]
```diff
@@ -45,76 +45,292 @@ To create an API key head to the <RegionalUrl text='Settings page' suffix='/sett
...
-        chatbot,
-        data=dataset,
-        evaluators=[is_concise],
-        experiment_prefix="my first experiment ",
-        max_concurrency=4,
-    )
 
+const client = new Client();
+
+const openai = new OpenAI();`,
```
Comment (baskaryan) [inline docs/evaluation/index.mdx:82]: to match python
```suggestion
const openaiClient = new OpenAI();`,
```

File: docs/evaluation/index.mdx [hunk 2]
```diff
@@ -45,76 +45,292 @@ To create an API key head to the <RegionalUrl text='Settings page' suffix='/sett
...
+# The SDK will automatically send the inputs from the dataset to your target function
+def target(inputs: dict) -> dict:
+    response = openai_client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[
+            { "role": "system", "content": "Answer the following question accurately" },
+            { "role": "user", "content": inputs["question"] },
+        ],
+    )
+    return { "response": response.choices[0].message.content.strip() }
```
Comment (baskaryan) [inline docs/evaluation/index.mdx:184]: python formatting nit
```suggestion
            {"role": "system", "content": "Answer the following question accurately"},
            {"role": "user", "content": inputs["question"]},
        ],
    )
    return {"response": response.choices[0].message.content.strip()}
```

File: docs/evaluation/index.mdx [hunk 3]
```diff
@@ -45,76 +45,292 @@ To create an API key head to the <RegionalUrl text='Settings page' suffix='/sett
...
+  `},
+    {
+      value: "typescript",
+      label: "TypeScript",
+      content: `// Define the application logic you want to evaluate inside a target function
+// The SDK will automatically send the inputs from the dataset to your target function
+async function target(inputs: string): Promise<{ response: string }> {
```
Comment (baskaryan) [inline docs/evaluation/index.mdx:191]: is inputs not a Record in js as well? didnt' realize you could have a target that just takes a string

File: docs/evaluation/index.mdx [hunk 4]
```diff
@@ -45,76 +45,292 @@ To create an API key head to the <RegionalUrl text='Settings page' suffix='/sett
...
+    response = openai_client.beta.chat.completions.parse(
+        model="gpt-4o-mini",
+        messages=[
+            { "role": "system", "content": instructions },
+            {
+                "role": "user",
+                "content": f"""Ground Truth answer: {reference_outputs["answer"]}; 
+                Student's Answer: {outputs["response"]}"""
+            },
+        ],
```
Comment (baskaryan) [inline docs/evaluation/index.mdx:238]: formatting nit
```suggestion
        to_evaluate = (
            f"Ground Truth Answer: {reference_outputs['answer']}\n"
            f"Student Answer: {outputs['response']}"
        )
        messages=[
            {"role": "system", "content": instructions},
            {"role": "user", "content": to_evaluate},
        ],
```

**Review by baskaryan — APPROVED** [review]

File: docs/evaluation/index.mdx (hunk 3 above)
Comment (davidx33) [reply docs/evaluation/index.mdx:191]: I just deconstructed it into a string before I sent it over to target

**Pull request merged**